catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
+elif [ -f /usr/bin/dnf ]
+then
+  sudo dnf install curl wget unzip git
+fi
+
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP || exit 1
+curl -s https://api.github.com/repos/PhantomGamers/SFP/releases/latest \
+| grep "SFP_UI-SelfContained-linux-x64.zip" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+unzip *.zip
+chmod +x SFP_UI
+./SFP_UI 
+
+# Install metro
+if [ -d ~/.steam ]
```

Comment:
```bash
steam_paths=("$HOME/.steam" "$HOME/.var/app/com.valvesoftware.Steam/.steam")

for path in "${steam_paths[@]}";
do
  [ -d "$path" ] && steam_path="$path"
done

while [ -z "$steam_path" ] || [ ! -d "$steam_path" ];
do
  echo ".steam directory not found. what is it's path?"
  read -r -p "--> " steam_path
done

install_path="$steam_path"/steam/skins/catppuccin
mkdir -p "$install_path"
git clone https://github.com/minischetti/metro-for-steam "$install_path"
```
no need to have pretty much the same commands twice (plus repos might be better as constants anyways just in case they ever change). having a fallback in case the steam directory would also be cool (probably with better wording though lol)

Reply:
lol good point